jkube-kit/common/src/main/java/org/eclipse/jkube/kit/common/util/FileUtil.java

```diff
@@ -360,5 +360,25 @@ public static void createDirectory(File directory) throws IOException {
             }
         }
     }
+
+    protected static String[] splitFilePathIntoParts(String filePath, String separator) {
+        // To deal with Windows Path separators we have to do this
```

Comment:
BTW, this comment is not accurate.
You have to re-escape the `\` character because `String#split` accepts a regex and `\ `is the escape sequence for Java regular expressions, so you have to re-escape it.